database/redis/build.gradle

```diff
@@ -0,0 +1,45 @@
+plugins {
+    id 'java'
+    id 'idea'
+    id 'io.quarkus'
+    id 'groovy'
+    id 'io.freefair.lombok' //version '6.1.0'
+}
+
+group 'com.example.jcc.database'
+
+repositories {
+    mavenLocal()
+    mavenCentral()
+}
+
+java {
+    sourceCompatibility = JavaVersion.VERSION_11
+    targetCompatibility = JavaVersion.VERSION_11
+}
+
+dependencies {
+    implementation enforcedPlatform(project(':bom'))
+
+    implementation project(':database:apid')
+    implementation project(':model')
+
+    implementation "io.quarkus:quarkus-reactive-pg-client"
```

Comment:
do we need to depend on postgres client ? It would be good to communicate with Postgres through API and leave it in postgres package. This could be raised as another issue.

Reply:
removed